### PR TITLE
Document alternative .terminal block syntax

### PR DESCRIPTION
**Pull request by Shredder121**
This has a few bonuses compared to manual `pre.terminal` elements.
It promotes readable authoring since code blocks are escaped automatically and the code examples are separated from the text.

### CODE REVIEW

**Comment by Shredder121** [comment]
I assumed it's okay to change the content in `changes/`.
If it's a no go then I'll undo that.

At least, I hope that from now on the content can be authored in the new format. :sparkles:

**Comment by kdaigle** [comment]
cc @gjtorikian do you have any thoughts on this change? :smile:

File: README.md [hunk 1]
```diff
@@ -77,12 +77,19 @@ Some actions return arrays.  You can modify the JSON by passing a block:
...
-Markdown could do this more cleanly.)
+You can specify terminal blocks by prefacing a [block element][block boundaries] with `{:.terminal}`.
+
+```markdown
+{:.terminal}
+    $ curl foobar
+```
+
+Alternatively, you can use plain html and use `pre.terminal` elements.
+(If you for example need specific syntax highlighting)
```
Comment (gjtorikian) [inline README.md]: I think we can change this parenthetical to read:

> If, for example, you need to emphasis text with `<em>`.

File: content/changes/2014-03-04-timezone-handling-changes.html [hunk 2]
```diff
@@ -30,6 +30,7 @@
 It is possible to supply a `Time-Zone` header which defines a timezone according
 to the [list of names from the Olson database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
 
+{:.terminal}
```
Comment (gjtorikian) [inline content/changes/2014-03-04-timezone-handling-changes.html]: :heart_eyes:

**Comment by gjtorikian** [comment]
I made a tiny comment on the README text, but it's a great change. Crazy to think that we didn't do this before. I especially like the addition of terminals to text that previously didn't have them.

**Comment by Shredder121** [comment]
Haha, thanks! I did indeed notice that there was a bad adoption of the terminal blocks.
Either the escaping made a lot look very ugly, or it was just not used.
I hope this helps at least from now on.

I'll address the comment on the README text.

**Comment by Shredder121** [comment]
Done.

File: content/changes/2014-03-04-timezone-handling-changes.html (hunk 2 above)
Comment (Shredder121) [inline content/changes/2014-03-04-timezone-handling-changes.html]: I knew it would be a good proposal. ;)

**Comment by kdaigle** [comment]
@Shredder121 thanks! :sparkles:

**Comment by Shredder121** [comment]
Nice! Thank you as well, happy coding.

**Comment by Shredder121** [comment]
Don't know if @kdaigle received the notification, but:
https://github.com/github/developer.github.com/commit/16f605ea6613f1707c6adb400cc11acf49db298f#commitcomment-13350791

There seems to be an ordering issue in the changes.atom feed.

Entries that have the same publish date seem to be randomly ordered.

**Pull request merged**